pkg/graph/resolvers/mto_solution.go

```diff
@@ -32,6 +32,11 @@ func MTOSolutionUpdate(
 	if err != nil {
 		return nil, fmt.Errorf("unable to update MTO Solution. Err %w", err)
 	}
+	// Clear out the name and type returned from the database for convenience so we don't try to set the custom name  or type fields
+	if existing.AddedFromSolutionLibrary() {
```

Comment:
Same here, if you wanna add the `logger.Warn()` from my suggestion above!

Reply:
Thanks @ClayBenson94, this is the same scenario, we expect this every time there is a solution sourced from a common solution.